<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="1" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
    </line>
  </reporter-output>
  <suite name="QAF BDD" duration-ms="47" started-at="2019-10-24T11:40:41Z" finished-at="2019-10-24T11:40:42Z">
    <groups>
    </groups>
    <test name="BDD Test" duration-ms="47" started-at="2019-10-24T11:40:41Z" finished-at="2019-10-24T11:40:42Z">
      <class name="com.qmetry.qaf.automation.step.client.DataDrivenScenario">
        <test-method status="PASS" signature="setupSuit(org.testng.ITestContext)[pri:0, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="setupSuit" is-config="true" duration-ms="7" started-at="2019-10-24T17:10:41Z" finished-at="2019-10-24T17:10:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@525f1e4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuit -->
        <test-method status="PASS" signature="setupTest(org.testng.ITestContext)[pri:0, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="setupTest" is-config="true" duration-ms="0" started-at="2019-10-24T17:10:41Z" finished-at="2019-10-24T17:10:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@525f1e4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="PASS" signature="setup(org.testng.ITestContext)[pri:0, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="setup" is-config="true" duration-ms="0" started-at="2019-10-24T17:10:41Z" finished-at="2019-10-24T17:10:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@525f1e4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="user successfully click on logout button( java.util.Map)[pri:1000, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="user successfully click on logout button" duration-ms="0" started-at="2019-10-24T17:10:41Z" finished-at="2019-10-24T17:10:41Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[com.qmetry.qaf.automation.testng.DataProviderException: Error while fetching data from ./resources/testData/login.csv]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: com.qmetry.qaf.automation.testng.DataProviderException: Error while fetching data from ./resources/testData/login.csv
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:111)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1020)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1110)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:781)
	at org.testng.TestRunner.run(TestRunner.java:635)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:122)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:172)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:104)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:70)
Caused by: com.qmetry.qaf.automation.testng.DataProviderException: Error while fetching data from ./resources/testData/login.csv
	at com.qmetry.qaf.automation.util.CSVUtil.getCSVDataAsMap(CSVUtil.java:186)
	at com.qmetry.qaf.automation.testng.dataprovider.QAFInetrceptableDataProvider.getData(QAFInetrceptableDataProvider.java:325)
	at com.qmetry.qaf.automation.testng.dataprovider.QAFInetrceptableDataProvider.interceptedDataProvider(QAFInetrceptableDataProvider.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:55)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	... 31 more
Caused by: org.json.JSONException: Bad character '' (6). at 6 [character 7 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:505)
	at org.json.CDL.rowToJSONArray(CDL.java:121)
	at com.qmetry.qaf.automation.util.StringUtil.getArrayFromCsv(StringUtil.java:559)
	at com.qmetry.qaf.automation.util.StringUtil.parseCSV(StringUtil.java:364)
	at com.qmetry.qaf.automation.util.CSVUtil.getCSVDataAsMap(CSVUtil.java:161)
	... 40 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- user successfully click on logout button -->
        <test-method status="PASS" signature="afterTest(org.testng.ITestContext)[pri:0, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="afterTest" is-config="true" duration-ms="0" started-at="2019-10-24T17:10:42Z" finished-at="2019-10-24T17:10:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@525f1e4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterClass(org.testng.ITestContext)[pri:0, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="afterClass" is-config="true" duration-ms="0" started-at="2019-10-24T17:10:42Z" finished-at="2019-10-24T17:10:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@525f1e4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterSuit(org.testng.ITestContext)[pri:0, instance:com.qmetry.qaf.automation.step.client.DataDrivenScenario@3e8]" test-instance-name="user successfully click on logout button" name="afterSuit" is-config="true" duration-ms="0" started-at="2019-10-24T17:10:42Z" finished-at="2019-10-24T17:10:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@525f1e4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuit -->
      </class> <!-- com.qmetry.qaf.automation.step.client.DataDrivenScenario -->
    </test> <!-- BDD Test -->
  </suite> <!-- QAF BDD -->
</testng-results>
